tests/codebuild/testfiles/xgboost-hosting-deployment-multi-container.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: sagemaker.aws.amazon.com/v1
+kind: HostingDeployment
+metadata:
+  name: xgboost-hosting-multi-container
+spec:
+  region: us-west-2
+  productionVariants:
+    - variantName: AllTraffic
+      modelName: xgboost-model
+      initialInstanceCount: 1
+      instanceType: ml.r5.large
+      initialVariantWeight: 1
+  models:
+    - name: xgboost-model
```

Comment:
we already have 2 hosting deployment tests with and without a name, can we add a container to either of them unless it breaks their assertions?

asking because hosting generates a lot of spam to logs